<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="whatsapp_lbl">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>WHATSAPP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_pb">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="send_pb">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>320</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mobile_le">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>140</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="mobile_lbl">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Enter name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="msg_lbl">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Enter message:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="msg_te">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>200</y>
      <width>311</width>
      <height>87</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="heading1_lbl">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Send to single person:</string>
    </property>
   </widget>
   <widget class="QLabel" name="heading2_lbl">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>410</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Send to multiple person:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="send2_pb">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>650</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLabel" name="limit_lbl">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>470</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Enter the limit:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="limit_le">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>470</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="msg2_te">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>530</y>
      <width>311</width>
      <height>87</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="msg2_lbl">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>530</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Enter message:</string>
    </property>
   </widget>
   <widget class="QLabel" name="msg_sent_lbl">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>710</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>939</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
